src/actions/login.js

```diff
@@ -8,48 +8,51 @@ import {
   UPDATE_MONEY,
   START_LOADING,
   FINISH_LOADING,
+  CATCH_API_ERROR,
 } from "../constants/actionTypes";
 
-export const saveUser = (result, user) => async (dispatch) => {
-  try {
-    if (result === "ok") {
-      dispatch({ type: LOGIN_SUCCESS, user });
-      return;
-    }
+const ERROR_MESSAGE = {
+  commonError: "유저 데이터 접근에 실패하였습니다.",
+};
```

Comment:
```suggestion
const ERROR_MESSAGE = {
 loginError: "유저 데이터 접근에 실패하였습니다.",
};
```
로그인 리덕스 스토어에도 에러 추가되면 공통 상수로 빼서 관리하는 것도 방법이겠네요.. 제가 나중에 수정하겠습니당